apps/easypid/src/features/share/FunkeRequestedAttributesDetailScreen.tsx

```diff
@@ -52,12 +52,39 @@ export function FunkeRequestedAttributesDetailScreen({
   const activeCredential = pidCredential?.id.includes(id) ? pidCredential : credential
 
   const [isSheetOpen, setIsSheetOpen] = useState(false)
+  const [elementPosition, setElementPosition] = useState(0)
+  const scrollViewRef = useRef<ScrollViewRefType>(null)
+
   useEffect(() => {
     navigation.setOptions({
       headerRight: () => <IconContainer icon={<HeroIcons.EllipsisHorizontal />} onPress={() => setIsSheetOpen(true)} />,
     })
   }, [navigation])
 
+  const toggleMetadataVisibility = () => {
```

Comment:
looks quite similar to on the other page, is there a risk between them getting out of sync? It's ok like this, just a though

Reply:
Created a reusable hook for it.